src/internal/partial_solution.rs

```diff
@@ -30,24 +30,43 @@ impl DecisionLevel {
 #[derive(Clone, Debug)]
 pub(crate) struct PartialSolution<DP: DependencyProvider> {
     next_global_index: u32,
+    /// The number of decisions that have been made, equal to the number of packages with decisions.
     current_decision_level: DecisionLevel,
```

Comment:
More than one decision can't be made per package?

Reply:
yes